app/scripts/modules/core/src/pipeline/config/stages/monitoreddeploy/DeploymentMonitorExecutionDetails.tsx

```diff
@@ -70,6 +73,11 @@ function getDeploymentMonitorAdditionalDetails(stage: IExecutionStage) {
   return null;
 }
 
+function isRunningOrComplete(stage: IExecutionStage) {
+  const runOrCompleteStatuses = ['RUNNING', 'SUCCEEDED', 'TERMINAL', 'FAILED_CONTINUE', 'PAUSED', 'CANCELED'];
```

Comment:
the `IExecutionStage` object has a few helpers you can use to check things like this — perhaps using those fields will help here (not sure, haven't checked closely): https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/src/orchestratedItem/orchestratedItem.transformer.ts#L61-L90